.gitignore

```diff
@@ -1,6 +1,13 @@
-logs
-orthoinference.out
-src/main/resources/*.txt
-src/main/resources/orthopairs/*
 *.txt
-target/*
\ No newline at end of file
+logs/
+target/
+
```

Comment:
I wonder if we should gitignore the default config.properties files in the release repos?

Reply:
Yeah, it would definitely be useful to ignore changes to the default `config.properties` so nothing gets committed to it accidentally.

I was using this method: https://stackoverflow.com/questions/3319479/can-i-git-commit-a-file-and-ignore-its-content-changes to keep the default `config.properties` in the repo but use a shell script that runs the "assume unchanged" command run on the default and creates a new copy of the `config.properties` file that is what is used by the program and has a name that is in .gitignore.  That is, have git overlook changes to the original and .gitignore the working copy.